src/charactersheet/viewmodels/character/magic_items/index.html

```diff
@@ -5,17 +5,13 @@
   <table class="table table-responsive table-ac-bordered table-hover">
     <thead>
       <tr class="ac-table-header">
-        <th class="col-xs-1" data-bind="click: function(){sortBy('attuned');}"> Attuned <span data-bind="css: sortArrow('attuned')"></span>
+        <th class="col-xs-9 col-sm-4 col-md-5 col-lg-4" data-bind="click: function(){sortBy('name');}"> Magic Item <span data-bind="css: sortArrow('name')"></span>
         </th>
-        <th data-bind="click: function(){sortBy('name');}"> Magic Item <span data-bind="css: sortArrow('name')"></span>
+        <th class="col-sm-6 text-center" colspan="3"
```

Comment:
Made all the charges bar be on column with colspan to look like all the other +/- bars